src/Diagnostics.DataProviders/KustoHeartBeatService/KustoHeartBeatService.cs

```diff
@@ -182,12 +182,20 @@ private async Task RunHeartBeatPrimary(string activityId)
                 _ConsecutiveSuccessCount = 0;
             }
 
-            // if not in failover state
-            //  should failover?
+            /*
+             * Logic of failing over to failover cluster:
```

Comment:
By failover cluster - you mean the main cluster?

Reply:
correct. 
The way this code is written is primary ==> follower, and Failover ==> main cluster